Source/Source/AdsEditorBinding.cs

```diff
@@ -132,11 +133,16 @@ public bool AreInterstitialsEnabled()
         {
             return true;
         }
-
+        
+        [Obsolete("Use \"void SetHasGDPRConsent(GdprConsent status)\" method instead.")]
         public void SetHasGDPRConsent(bool consent)
         {
         }
 
+        public void SetHasGDPRConsent(GdprConsent consent)
+        {
```

Comment:
We may need to log something here, so at least we have some feedback on the editor

Reply:
Done!